site/src/pages/components/selectmenu.mdx

```diff
@@ -378,3 +379,14 @@ Like other form control elements, selectmenu supports validity checking attribut
 * [HTML spec for `checkValidity()`](https://html.spec.whatwg.org/#dom-cva-checkvalidity)
 * [HTML spec for `reportValidity()`](https://html.spec.whatwg.org/#dom-cva-reportvalidity)
 * [HTML spec for `setCustomValidity()`](https://html.spec.whatwg.org/#dom-cva-setcustomvalidity)
+
+## Why not improve `<select>`
+
+Instead of creating a new `<selectmenu>` element, we could improve the existing select element to meet the same needs. However, there are a few problems with this (borrowed from the [Custom Controls UI Explainer](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/ControlUICustomization/explainer.md):
+
+- Changing the parsing behavior for `<select>` to allow other element children by default would likely break a lot of old sites that depend on the current behavior.
```

Comment:
Are there any references to attempts to validate or quantify the back-compat concern here? HTTP Archive queries for `<select>` containing children other than `<option>` perhaps?

Reply:
@justinfagnani this is a valid ask and no we didn't concretely look into an enumeration of potential content that would be impacted. I presume you're asking from a PE perspective as the parsing changes to support arbitrary content is non-trivial and has security issues since the current select popup escapes window bounds.

Reply:
@bkardell you're a wiz with HTTP archive, any chance you can do a query to derive non option or optgroup children of a select?